src/main/kotlin/no/nav/medlemskap/regler/v2/arbeidsforhold/RegelgruppeForArbeidsforhold.kt

```diff
@@ -0,0 +1,147 @@
+package no.nav.medlemskap.regler.v2.arbeidsforhold
+
+import no.nav.medlemskap.domene.Arbeidsforholdstype
+import no.nav.medlemskap.domene.Skipsregister
+import no.nav.medlemskap.regler.common.Funksjoner.erDelAv
+import no.nav.medlemskap.regler.common.Funksjoner.erIkkeTom
+import no.nav.medlemskap.regler.common.Personfakta
+import no.nav.medlemskap.regler.common.Resultattype.*
+import no.nav.medlemskap.regler.v2.common.*
+
+class RegelgruppeForArbeidsforhold(val personfakta: Personfakta) : Regelgruppe() {
+
+    override fun evaluer() =
+            Spørsmål(
+                    identifikator = "1.5",
+                    spørsmål = "Oppfyller arbeidsforholdet kravet?",
+                    beskrivelse = "",
+                    operasjon = { kjørRegler() }
+            ).utfør()
+
+
+    private fun kjørRegler(): Svar {
+        val arbeidsforhold = harArbeidsforhold.utfør()
+
+        if (arbeidsforhold.svar.resultat == NEI) {
+            return Svar(
+                    resultat = UAVKLART,
+                    begrunnelse = "Brukeren mangler registrerte arbeidsforhold",
+                    underspørsmål = listOf(arbeidsforhold)
+            )
+        }
+
+        val norskArbeidsgiver = erArbeidsgiverNorsk.utfør()
+
+        if (norskArbeidsgiver.svar.resultat == NEI) {
+            return Svar(
+                    resultat = UAVKLART,
+                    begrunnelse = "Brukeren har ikke norsk arbeidsgiver",
+                    underspørsmål = listOf(arbeidsforhold, norskArbeidsgiver)
+            )
+        }
+
+        val maritimt = erArbeidsforholdetMaritimt.utfør()
+
+        if (maritimt.svar.resultat == NEI) {
+            val kabinansatt = erPersonenPilotEllerKabinansatt.utfør()
+
+            if (kabinansatt.svar.resultat == JA) {
+                return Svar(
+                        resultat = UAVKLART,
+                        begrunnelse = "Brukeren er pilot eller kabinansatt og må utredes nærmere",
+                        underspørsmål = listOf(arbeidsforhold, norskArbeidsgiver, maritimt, kabinansatt)
+                )
+            }
+
+            return Svar(
+                    resultat = JA,
+                    begrunnelse = "Krav oppfylt",
+                    underspørsmål = listOf(arbeidsforhold, norskArbeidsgiver, maritimt, kabinansatt)
+            )
+        } else {
```

Comment:
Dette er mer "vanlig kode" enn det var før, synes jeg. Return-statements, if-else. Jeg tror dette kanskje er mindre lesbart for Helle enn det var før (?). Som nevnt tidligere, så bør vi kanskje avklare hvem som skal kunne forstå denne koden..?